### ISSUE TITLE
DetailsList: Screen reader doesn't announce selected row number

### ISSUE THREAD

**Issue by sereymonyros** [microsoft/fluentui#16115]
<DetailsList />

Step to prop: use narrator to go detail into children of the list, narrator doesn't announce number of the row is selected

Actual result: narrator doesn't announce number of row selected, so user doesn't know where they are up to
Expected result: narrator should announce so user is able to know where they are up to in the list

Thanks,

**Comment by ecraig12345** [comment]
Thanks for reporting this, though next time you file an issue, please fill out the whole template (including a repro link and the specific library version you're using) instead of deleting it.

I can repro this on the [main DetailsList example](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist#usage), but I think it might be a bug with Narrator itself. We're applying [`aria-rowindex`](https://www.w3.org/TR/wai-aria-1.1/#aria-rowindex) to each DetailsRow (on the `role="row"` element) and `aria-rowcount` to the parent DetailsList (on the `role="grid"` element), which as far as I can tell is the correct markup to inform Narrator of the row numbers.

@paulgildea or @JustSlone, what's our current process for reporting bugs to the Narrator team?

**Comment by ecraig12345** [comment]
Other investigation:
- Behavior is similar on a DetailsList example with or without selection enabled (the current row number is not read regardless)
- The closest ARIA practices example I could find is [this data grids example](https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html), and it does read the row number, but there are substantial differences that make it not a great comparison with DetailsList (it uses actual `<table>` elements, appears to select a single cell at a time not the whole row, and is not interactive in the sense that the rows are selectable)
- NVDA behavior:
  - Firefox: works properly (reads row number)
  - Edge (new): doesn't work
  - Chrome: doesn't work

**Comment by paulgildea** [comment]
@ecraig12345 We have to contact their team internally. I don't believe they have a GitHub issue filing. Let's follow up if we believe we're at spec and the issue is with the tooling.

**Comment by bcoard** [comment]
@ecraig12345 Is this the same issue as https://github.com/microsoft/fluentui/issues/15355?

**Comment by ecraig12345** [comment]
@bcoard I think the other one is specific to lists with grouping (whereas this one repros without groups).

**Comment by bcoard** [comment]
@ecraig12345 Ah, ok thanks! I didn't know if the root cause was essentially the same or not :)

**Comment by ecraig12345** [comment]
@bcoard Might be, but I think it's worth tracking separately

**Comment by jimmy3912msncom** [comment]
Any status on the investigation? We are also seeing the same issue and are using 7.153.2 of office-ui-fabric-react for the DetailsList. We are seeing the same behavior described in the thread

**Comment by JustSlone** [comment]
We're investigating some other DetailsList issues right now. A lot of times these are not simple fixes. Assigning this one to @smhigley since she's looking into other related issues. We'll need to go through the other issues first though.

**Comment by smhigley** [comment]
The root cause of this is putting focus on row elements instead of cells, which is the expected interaction for grids. I don't think this will be resolved until we can overhaul the DetailsList focus/navigation behavior.
